src/main/resources/add-ons-to-index.json

```diff
@@ -4471,6 +4471,23 @@
         "package": "emrmonitor"
       }
     },
+    {
+      "uid": "org.openmrs.omod.facelist",
+      "type": "OMOD",
+      "name": "FACE LIST MODULE",
+      "maintainers": [
+        {
+          "name": "Mutesasira Moses",
+          "url": "https://github.com/mozzy11"
+        }
+      ],
+      "backend": "org.openmrs.addonindex.backend.Bintray",
+      "bintrayPackageDetails": {
+        "owner": "openmrs",
+        "repo": "omod",
+        "package": "openmrs-module-facelist"
```

Comment:
Also, can you modify the package name to facelist (We can confirm this only once it has been added to the OpenMRS repo on Bintray but it mostly should be this)